### ISSUE TITLE
[10.1.1] Electrical Water Pump output capacity does not match file value

### ISSUE THREAD

**Issue by Praefator** [StrangeLoopGames/EcoIssues#24665]
As per title. Possible relation to #16962 
\Mods\__core__\AutoGen\WorldObject\ElectricWaterPump.cs defines the output capacity as follows:

``this.GetComponent<LiquidProducerComponent>().Setup(typeof(WaterItem), 4, BlockOccupancyType.WaterOut);``

\Mods\__core__\AutoGen\WorldObject\MechanicalWaterPump.cs meanwhile defines the output capacity as follows:

`` this.GetComponent<LiquidProducerComponent>().Setup(typeof(WaterItem), 1, BlockOccupancyType.WaterOut);``

In other words, theoretically the mechanical water pump should output 1 water/sec, while the electrical water pump should output **4** water/sec. In reality, both output 1 water/sec, as evidenced by the screenshots:

![20240121004301_1](https://github.com/StrangeLoopGames/EcoIssues/assets/841990/96810a2d-3a87-46a3-b1a5-fa9c5239a44d)
![20240121004329_1](https://github.com/StrangeLoopGames/EcoIssues/assets/841990/39e25a1b-8461-47a2-863c-6a25ab794898)

**Comment by D3nnis3n** [comment]
Was fixed in Update 10.2.
